scheduler/scheduler_sync.daemon

```diff
@@ -103,14 +110,22 @@ do
                 # Verify md5sum of destination file is same as source
                 md5sum --check --status $HOME/tmp.md5
                 retval=$?
-                if [[ $retval -ne 0]]; then
+                rm $HOME/tmp.md5
+
+                if [[ $retval -ne 0 ]]; then
                     printf "$(date --utc "+%Y%m%d %H:%M:%S UTC") - Error in syncing schedule, trying again\n"
-                    sleep 60  # Try again in 60s
+                    attempts=$((attempts + 1))
+                    if [[ $attempts -ge $MAX_ATTEMPTS ]]; then
```

Comment:
This will continually send messages to the slack channel right? What if you just sent one ($attemps -eq $MAX_ATTEMPTS or something) and then a new message when it works again?

Reply:
Good catch, I'll fix that. The other alert script only sends one message using your suggested logic as well.